# скорость вперед
float32 velocity_forward

# скорость вправо
float32 velocity_right

# скорость вниз
float32 velocity_down